pages/expense/[id].jsx

```diff
@@ -30,7 +31,7 @@ function DetailPage({ setToast }) {
     return;
   }
 
-  return <ExpenseDetail expense={data} setToast={setToast} />;
+  return <ExpenseDetail expense={data} setToast={setToast} expid={id} />;
```

Comment:
Again, you could alias the `data` property:

```
  const { data: expense, isLoading, error } = useSWR(`/api/expenses/${id}`);
```

```suggestion
  return <ExpenseDetail expense={expense} setToast={setToast} expenseId={id} />;
```
And also here: do you really need an extra prop `expenseId`?